<?xml version="1.0"?>

<launch>

	<arg name="marker_size" default="0.04" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/narrow_stereo/left/image_mono" />
	<arg name="cam_info_topic" default="/narrow_stereo/left/camera_info" />	
	<arg name="output_frame" default="/torso_lift_link" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />

	<node name="apc_marker" pkg="apc_marker" type="apc_marker_node" output="screen"/>
	
</launch>
